.github/workflows/trigger-ui-tests.yml

```
@@ -24,7 +24,7 @@ jobs:
 
           git fetch --prune origin
           BRANCHES=$(git ls-remote --heads origin | awk -F'/' '{print $3"/"$4}' | grep '^customer/')
-          SKIP_BRANCHES=("customer/sav" "customer/vsb-tuo" "customer/zcu-data")
+          SKIP_BRANCHES=("customer/sav" "customer/vsb-tuo")
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)